packages/tinacms/src/components/modals/ModalProvider.tsx

```diff
@@ -110,14 +111,49 @@ export const ModalOverlay = styled.div`
   padding: 0;
 `
 
-export const ModalHeader: StyledComponent<'div', {}> = styled.div`
+const ModalTitle = styled.h2`
+  font-size: ${font.size(5)};
+  font-weight: 500;
+  line-height: normal;
+  margin: 0;
+`
+
+const CloseButton = styled.div`
+  display: flex;
+  align-items: center;
+  fill: ${color.grey(5)};
+  cursor: pointer;
+  transition: fill 85ms ease-out;
+  svg {
+    width: 1.5rem;
+    height: auto;
+  }
+  &:hover {
+    fill: ${color.grey(8)};
+  }
+`
+
+interface ModalHeaderProps {
+  title: string
+  close(): void
+}
+
+export const ModalHeader = styled(
+  ({ title, close, ...styleProps }: ModalHeaderProps) => {
```

Comment:
What do you think of using `children` instead of `title`:

```jsx
<ModalHeader close={close}>Reset</ModalHeader>
```

Reply:
Ah, that's smart & cleans it up a bit. I'll make that change.